bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -378,6 +382,10 @@ impl<'de> serde::Deserialize<'de> for OutputDto {
                     ExtendedOutputDto::deserialize(value)
                         .map_err(|e| serde::de::Error::custom(format!("can not deserialize output: {}", e)))?,
                 ),
+                AliasOutput::KIND => OutputDto::Alias(
+                    AliasOutputDto::deserialize(value)
+                        .map_err(|e| serde::de::Error::custom(format!("can not deserialize output: {}", e)))?,
```

Comment:
Maybe also add the output type in the message? (also missing in the other error messages)